spring-data-r2dbc/src/main/java/org/springframework/data/r2dbc/mapping/R2dbcMappingContext.java

```diff
@@ -25,14 +25,15 @@
  * R2DBC-specific extension to {@link RelationalMappingContext}.
  *
  * @author Mark Paluch
+ * @author Jens Schauder
  */
 public class R2dbcMappingContext extends RelationalMappingContext {
 
 	/**
 	 * Create a new {@link R2dbcMappingContext}.
 	 */
 	public R2dbcMappingContext() {
-		setForceQuote(false);
+		setForceQuote(true);
```

Comment:
`setForceQuote(…)` could be removed entirely as it sets the value to its default value, see `JdbcMappingContext`.

Reply:
done